src/features/ci-cd/hooks/track-latest-release.ts

```diff
@@ -107,24 +116,23 @@ const updateLatestRelease = async (
 hooks.addPureHook('PATCH', 'resin', 'release', {
 	POSTRUN: async (args) => {
 		const { request } = args;
-		// If we're updating a build by id and setting it successful then we update the application to this build
+		// If we're updating a release by id and setting it successful then we update the application to this build
 		if (request.odataQuery != null) {
 			const keyBind = request.odataQuery.key;
 			// TODO: Support named keys
 			if (keyBind != null && 'bind' in keyBind) {
-				const id = sbvrUtils.resolveOdataBind(request.odataBinds, keyBind);
-				await updateLatestRelease(id, args);
+				await updateLatestRelease(args);
 			}
 		}
 	},
 });
 
 hooks.addPureHook('POST', 'resin', 'release', {
 	POSTRUN: async (args) => {
-		// If we're creating a build then check if the latest release needs to be updated
+		// If we're creating a release then check if the latest release needs to be updated
 		const id = args.result;
 		if (id != null) {
```

Comment:
There's no need to store to a var now and slightly changing the comment to explain why we have this check makes sense imo - ie don't bother checking for release updates if we didn't actually create anything
```suggestion
		// If we succesfully created a release then check if the latest release needs to be updated
		if (args.result != null) {
```